content/en/docs/concepts/configuration/secret.md

```diff
@@ -102,19 +102,23 @@ See [decoding a secret](#decoding-a-secret) for how to see the contents.
 
 #### Creating a Secret Manually
 
-You can also create a secret object in a file first,
-in json or yaml format, and then create that object.
+You can also create a secret in a file first, in json or yaml format,
+and then create that object. The secret contains two maps: data and
+stringData. The data map is used to store arbitrary data, encoded using base64.
```

Comment:
The Secret contains -- Here you are referring to the Kubernetes Secret resource that contains two fields, data and stringData, correct? Possibly add a link to the API resource to avoid confusion,
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#secret-v1-core

Reply:
@kbhawkey that makes sense - I've capitalised Secret, added a link to the reference docs, and I've also tried to sort a few other places that weren't capitalised.